src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java

```diff
@@ -81,30 +81,18 @@ public OctoChannelAllocator(
 
         boolean useIce = false;
         boolean useDtls = false;
-        boolean useRtx = false; // config.isRtxEnabled();
-        boolean enableRemb = false; // config.isRembEnabled();
-        boolean enableTcc = false; // config.isTccEnabled();
-        boolean useSctp = false; // config.openSctp() == null || config.openSctp();
```

Comment:
Can we get away with removing the code that supports this?  Previously we said we didn't want to remove it from the bridge because of people's deployments.

Reply:
We haven't migrated to websockets everywhere AFAIK

Reply:
Then how can we disable support for using SCTP?  (Or perhaps I've misunderstood the implication of removing this flag--I assumed it meant we never would do sctp)

Reply:
This is the `OctoChannelAllocator`.  We still conditionally add SCTP for the `ParticipantChannelAllocator`.

Reply:
Ah! Missed that.